python/graphstorm/gconstruct/transform.py

```diff
@@ -454,11 +454,21 @@ class NumericalMinMaxTransform(TwoPhaseFeatTransform):
     out_dtype:
         The dtype of the transformed feature.
         Default: None, we will not do data type casting.
+    transform_conf : dict
```

Comment:
i suggest to pass min_val and max_val directly. in this way, the doc string can explain what are these two values and what is their type.